src/pubtools/_marketplacesvm/arguments.py

```diff
@@ -181,3 +181,28 @@ def __call__(self, parser, namespace, values, options=None):
             if not isinstance(items, list):
                 raise ArgumentError(self, f"Expected value to be a list, got: {type(items)}")
         setattr(namespace, self.dest, items)
+
+
+class RepoFileQueryLoad(Action):
+    """
+    Argparse Action subclass for loading StArMap mappings from the ``repo-file`` argument.
+
+    This action is intended to allow the optional load of mappings from a json file
+    instead of having to request data from server.
+
+    It will evaluate the input file and set it as a StArMap's QueryResponse.
```

Comment:
Oops! Rigt now we're using `QueryResponseContainer` from `APIv2`, I forgot to update the docs from the `RepoQueryLoad` action, my bad!

When I did the APIv2 changes back then I've changed the action to convert just into a simple JSON and then it would get loaded to `QueryResponseContainer` format later on here: https://github.com/release-engineering/pubtools-marketplacesvm/blob/main/src/pubtools/_marketplacesvm/services/starmap.py#L67-L80

Can you please update them all saying it's a JSON in the `QueryResponseContainer` format? 

Here is the doc JFTR: https://release-engineering.github.io/starmap-client/model/models.html#apiv2

Reply:
Can do!

Reply:
TYVM